Loop54.Tests.Shared/CallMethods.cs

```diff
@@ -45,8 +45,6 @@ public void SearchHasResults([Values("steak", "chicken breast")]string query)
             var response = GetClient().Search(WrapRequest(new SearchRequest(query)));
             Assert.Greater(response.Results.Count, 0);
             Assert.Greater(response.Results.Items.Count, 0);
-            Assert.Greater(response.RelatedResults.Count, 0);
```

Comment:
In the current engine version we no longer return related results for the test queries, as we now tend to merge related results into "direct" results (so the change is intentional).